post.hbs

```diff
@@ -31,8 +21,28 @@
                             </span>
                         {{/foreach}}
                     </span>
+                    <span>
+                        Reading Time:&nbsp;
```

Comment:
I think this isn't necessary, "3 Min Read" should be clear enough.

Reply:
Done